src/main/java/duke/Event.java

```diff
@@ -0,0 +1,28 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+
+public class Event extends Task {
+    protected LocalDate at;
+
+    public Event(String description, LocalDate at) {
+        super(description);
+        this.at = at;
+    }
+    
+    private String getFormattedDate() {
```

Comment:
Perhaps you can consider building a `Parser` and let him handle the _formatting_ and _converting to file string_ issues? 😃